components/tikv_util/src/config.rs

```diff
@@ -271,6 +276,114 @@ impl<'de> Deserialize<'de> for ReadableSize {
     }
 }
 
+pub mod readable_size_in_decimal_base_serde {
```

Comment:
Can the code be merged with 2 based unit?

Reply:
I can refactor a bit, but they can't be merged as one implementation. That would means changing the default interpretation for `MB/GB`, which leads to 1) old configuration item will be interpreted differently in newer binary 2) in the future user has to explicitly use `MiB/GiB`